packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6155,6 +6155,8 @@ export default class Meeting extends StatelessWebexPlugin {
           // so reset the timer as it's not needed anymore, we want to reconnect immediately
           this.reconnectionManager.resetReconnectionTimer();
 
+          this.cleanUpBeforeReconnection();
```

Comment:
`Meeting.reconnect()` is also called from some other places and I think it would be good to call `cleanUpBeforeReconnection()` for those case too, so maybe better to just call it from inside `Meeting.reconnect()` so that it's always done when we reconnect?

Reply:
Done